### PR TITLE
docs(getting started) Update Getting Started landing page

### PR DESCRIPTION
**Pull request by maggiehays**
This PR contains the following:

* Update to existing "Get Started" landing page to provide high-level overview of the section
* Update quicklinks

## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: docs/get-started-with-datahub.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+## Get Started with Self-Hosted DataHub
+
+This is a great place to start if you're new to DataHub and are interested in taking the open-source project for a spin.
+
+You'll learn how to:
+
+* Deploy DataHub on your local machine
+* Ingest sample data to get a feel for what is possible in DataHub
+* Add and invite users to DataHub
+* Troubleshoot common issues
```
Comment (maggiehays) [inline docs/get-started-with-datahub.md:14]: We'll need to update this list as the scope of the section expands

File: docs/get-started-with-datahub.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## Get Started with Managed DataHub
+
+[Acryl Data](https://www.acryldata.io/product) provides a fully managed, premium version of DataHub.
+
+In this section, you'll learn how to:
+
+* Add and invite users to DataHub
+* Configure Slack notifications
+* Understand Approval Workflows
```
Comment (maggiehays) [inline docs/get-started-with-datahub.md:47]: Same as above -- We'll need to update this list as the scope of the section expands

File: docs/get-started-with-datahub.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Before you go further, you'll need to have a DataHub instance provisioned. The Acryl Integrations Team will provide you with the following once it has been deployed:
+
+* The URL for your Acryl instance, i.e. `https://<your-domain-name>.acryl.io`
+* Admin account credentials for logging into the DataHub UI
+
+<p>
+<a
+    className='button button--primary button--lg'
+    href="authentication/guides/add-users">
```
Comment (maggiehays) [inline docs/get-started-with-datahub.md:59]: This hyperlinks out to the first article in the Managed DataHub section; we'll need to update the link if we change the ordering of articles in the sidebar.

File: docs/get-started-with-datahub.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+Make sure to allocate enough hardware resources for Docker! 
+We recommend the following configuration: 2 CPUs, 8GB RAM, 2GB Swap area, and 10GB disk space.
+
+:::
+
+<p>
+<a
+    className='button button--primary button--lg'
+    href="quickstart">
```
Comment (maggiehays) [inline docs/get-started-with-datahub.md:34]: This hyperlinks out to the first article in the Self-Hosted DataHub section; we'll need to update the link if we change the ordering of articles in the sidebar and/or if we split apart Quickstart.

File: docs/get-started-with-datahub.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
+# Get Started With DataHub
```
Comment (maggiehays) [inline docs/get-started-with-datahub.md:1]: The main purpose of this document is to provide the "Get Started" section with a landing page of sorts, so it's intentionally sparse.

File: docs/get-started-with-datahub.md (hunk 1 above)
Comment (aditya-radhakrishnan) [reply docs/get-started-with-datahub.md:14]: Good point!

File: docs/get-started-with-datahub.md (hunk 2 above)
Comment (aditya-radhakrishnan) [reply docs/get-started-with-datahub.md:47]: Should we add set up SSO here?

Comment (aditya-radhakrishnan) [reply docs/get-started-with-datahub.md:47]: As an alternative to add and invite users

File: docs/get-started-with-datahub.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+* Deploy DataHub on your local machine
+* Ingest sample data to get a feel for what is possible in DataHub
+* Add and invite users to DataHub
+* Troubleshoot common issues
+
+### Prerequisites
+
+Before you go further, ensure you have the following installed:
+
+* [Docker](https://docs.docker.com/get-docker/)
```
Comment (aditya-radhakrishnan) [inline docs/get-started-with-datahub.md:20]: We also need Python right?

Comment (maggiehays) [reply docs/get-started-with-datahub.md:20]: Ah good point - what version do we require?

File: docs/get-started-with-datahub.md (hunk 2 above)
Comment (maggiehays) [reply docs/get-started-with-datahub.md:47]: @aditya-radhakrishnan we haven't migrated SaaS-specific SSO configuration docs yet, so I'm a little hesitant to add it in until we do

Comment (aditya-radhakrishnan) [reply docs/get-started-with-datahub.md:48]: Makes sense!

File: docs/get-started-with-datahub.md (hunk 6 above)
Comment (aditya-radhakrishnan) [reply docs/get-started-with-datahub.md:21]: Looks good!

**Review by aditya-radhakrishnan — APPROVED** [review]

**Pull request merged**